test/evaluation/selectFrom.spec.ts

```diff
@@ -2329,7 +2329,7 @@ describe('selectFrom tests', () => {
               id: 'did:example:ebfeb1f712ebc6f1c276e12ec21',
             },
             id: 'https://eu.com/claims/DriversLicense',
-            issuanceDate: '2010-01-01T19:73:24Z',
+            issuanceDate: '2010-01-01T19:23:24.000Z',
```

Comment:
Why are we changing verifiable credentials invalidating the associated proofs?

Reply:
Fixed!